spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/amqp/RabbitProperties.java

```diff
@@ -315,6 +316,11 @@ public Template getTemplate() {
 		private String keyStore;
 
 		/**
+		 * Set the key store type. See {@link RabbitConnectionFactoryBean#getKeyStoreType()}
```

Comment:
Could you remove the second sentence please? We don't use Javadoc links in the field javadoc for configuration properties as it messes up the auto-generated metadata used by your IDE.

Reply:
Cheers Andy. Good point by the way. Done!